UITests/Global.swift

```diff
@@ -533,9 +533,9 @@ class SearchUtils {
         return mockSearchEngine("YouTube", template: "https://m.youtube.com/#/results?q={searchTerms}&sm=", icon: "youtube")!
     }
 
-    static func mockSearchEngine(name: String, template: String, icon: String) -> OpenSearchEngine? {
-        guard let imagePath = NSBundle(forClass: self).pathForResource(icon, ofType: "ico"),
-              let imageData = NSData(contentsOfFile: imagePath),
+    static func mockSearchEngine(_ name: String, template: String, icon: String) -> OpenSearchEngine? {
+        guard let imagePath = Bundle(for: self).path(forResource: icon, ofType: "ico"),
+              let imageData = Data(contentsOfFile: imagePath),
```

Comment:
here I get "Argument labels '(contentsOfFile:)' do not match any available overloads" error. would something like below work instead?
guard let imagePath:String = Bundle(for: self).path(forResource: icon, ofType: "ico"),
            let image = UIImage(contentsOfFile: imagePath)
@farhanpatel ?